.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:8
+      - image: circleci/node
```

Comment:
Is it safe to assume that circleci defaults to lts?

Reply:
The docker image defaults to `latest`. Let me change that to `10`.